settings.py

```diff
@@ -0,0 +1,14 @@
+from pydantic.v1 import BaseSettings
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "FastAPI weather temperature management"
+
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./weather_data.db"
+
+    class Config:
+        case_sensitive = True
+        env_file = ".env"
```

Comment:
The `env_file` is set to `.env`, which means environment variables can be loaded from this file. Ensure that the `.env` file exists and contains the necessary configurations for your application.